app.js

```diff
@@ -0,0 +1,20 @@
+require('dotenv').config();
+const express = require('express');
+const todoRouter = require('./router');
+const usersRouter = require('./router/usersRouter.js');
+const errorHandler = require('./middleware/errorHandler.js');
+
+const app = express();
+const port = 3000;
+
+app.use(express.json());
+app.use(express.urlencoded({extended: true}));
+
+app.get('/', (req, res) => {
+  res.send('Hello World');
+})
+app.use('/', usersRouter);
```

Comment:
disini ntar tinggal use 
```suggestion
app.use(routes);
```